requests/models.py

```diff
@@ -796,7 +800,23 @@ def iter_lines(self, chunk_size=ITER_CHUNK_SIZE, decode_unicode=None, delimiter=
             if delimiter:
                 lines = chunk.split(delimiter)
             else:
+                # Python splitlines() supports the universal newline (PEP 278).
+                # That means, '\r', '\n', and '\r\n' are all treated as end of 
+                # line. If the last chunk ends with '\r', and the current chunk
+                # starts with \n, they should be merged and treated as only 
```

Comment:
Minor nit: as there are inverted commas around `'\r'` in the line above, they should be added here around `\n`.